cmd/bosun/web/web.go

```diff
@@ -390,7 +392,7 @@ func JSON(h func(miniprofiler.Timer, http.ResponseWriter, *http.Request) (interf
 		}
 		buf := new(bytes.Buffer)
 		enc := json.NewEncoder(buf)
-		if strings.Contains(r.Header.Get("Accept"), "html") {
+		if strings.Contains(r.Header.Get("Accept"), "html") || strings.Contains(r.Host, "localhost") {
```

Comment:
What does this change do?

Reply:
that makes json render pretty if you are running locally, but compact anywhere else. It makes chrome debugger readable.